server/impl/meetup.go

```diff
@@ -298,10 +299,7 @@ func (i *Implementation) PostMeetup(params operations.PostMeetupParams, _ interf
 	}
 
 	modelMeetup := modelMeetupRequestBodyToModelMeetup(params.Meetup, id.(string))
-	if err := i.modelMeetupToDBMeetup(&dbMeetup, &modelMeetup); err != nil {
-		logger.WithError(err).Error("Failed to create db meetup object")
-		return responders.InternalServerError{}
-	}
+	_ = i.modelMeetupToDBMeetup(&dbMeetup, &modelMeetup)
```

Comment:
modelMeetupToDBMeetup never returns an error

Reply:
nice catch, should we also update the function signature then? I think it still says it returns an error